roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -99,15 +99,14 @@ policyConfig:
   bootstrapPolicyFile: {{ openshift_master_policy }}
   openshiftInfrastructureNamespace: openshift-infra
   openshiftSharedResourcesNamespace: openshift
-{# TODO: Allow users to override projectConfig items #}
 projectConfig:
   defaultNodeSelector: "{{ openshift.master.default_node_selector | default("") }}"
-  projectRequestMessage: ""
-  projectRequestTemplate: ""
+  projectRequestMessage: "{{ openshift.master.project_request_message | default("") }}"
+  projectRequestTemplate: "{{ openshift.master.project_request_template | default("") }}"
   securityAllocator:
-    mcsAllocatorRange: s0:/2
-    mcsLabelsPerProject: 5
-    uidAllocatorRange: 1000000000-1999999999/10000
+    mcsAllocatorRange: "{{ openshift.master.project_request_template | default("s0:/2") }}"
+    mcsLabelsPerProject: "{{ openshift.master.mcs_labels_per_project | default("5") }}"
+    uidAllocatorRange: "{{ openshift.master.uid_allocator_range | default("1000000000-1999999999/10000") }}"
```

Comment:
Do you want me to put this on the openshift_facts for now? Looks pretty easy to do .